src/registry.ts

```diff
@@ -143,5 +156,13 @@ export async function setEntry(
     ...opts,
     method: "post",
     data,
+    // Transform the request to remove quotes, since the revision needs to be
+    // parsed as a uint64 on the Go side.
+    transformRequest: function (data: unknown) {
+      // Convert the object data to JSON.
+      const json = JSON.stringify(data);
+      // Change the revision value from a string to a JSON integer.
+      return json.replace(/"revision":"([0-9]+)"/, '"revision":$1');
```

Comment:
We have this regex in more than one spot. This is a possible place to introduce a bug. Let's move it to a const and use that instead.

Reply:
It's actually a different regex, one has quotes around the revision value and one doesn't

Reply:
Ah, right, this one basically does the reverse. OK but I would still feel better if those are in named constants. Especially if they are so visually similar.

Reply:
Just saw this comment. Okay, will do.